dhis-2/dhis-support/dhis-support-sql/src/main/java/org/hisp/dhis/db/sql/AbstractSqlBuilder.java

```diff
@@ -303,4 +303,17 @@ protected String notSupported() {
   protected <T> String toCommaSeparated(Collection<T> collection, Function<T, String> mapper) {
     return collection.stream().map(mapper).collect(Collectors.joining(","));
   }
+
+  /**
+   * Checks if the given input is quoted.
+   *
+   * @param input the input string.
+   * @return true if the input is quoted, false otherwise.
+   */
+  protected static boolean isQuoted(String input) {
```

Comment:
Probably this method can be moved to org.hisp.dhis.commons.util.TextUtils.java

Or the logic could be extracted to that class (TextUtils) and invoked from this method.